editor/plugins/theme_editor_plugin.cpp

```diff
@@ -3664,6 +3665,59 @@ void ThemeEditor::_preview_control_picked(String p_class_name) {
 	theme_type_editor->select_type(p_class_name);
 }
 
+bool ThemeEditor::can_drop_data_fw(const Point2 &p_point, const Variant &p_data, Control *p_from) {
+	Dictionary d = p_data;
```

Comment:
```suggestion
	const Dictionary d = p_data;
```
Same for Vector and String.